### DISCUSSION TITLE
TC Pairs files not loading into database

### DISCUSSION THREAD

**Discussion by mollybsmith-noaa** [dtcenter/METplus#2461] [File I/O] [answered]
Hi,

Ever since I upgraded to METplus 5.1.0 and MET 11.1.0 around September 10th, I haven't been able to successfully load any TC Pairs output into the METplus mysql database. I'm not sure what is going wrong, as I've run the sql update scripts every time there is a release, and met_db_load.py doesn't seem to throw any errors, but the stats never appear in the database. Could you look into this?

The TC Pairs output is on Jet at /lfs1/BMC/amb-verif/tcmet/output/tc_pairs_post/nhc/GFSO. Files before September, which were run with 4.0.1 and 10.0.1, load just fine, but files after September do not.

I'm using METdataio 2.1.0, with the database loading portion cloned into /home/role.amb-verif/GSL_verif-global/bin. These same scripts load GridStat, PointStat, and MODE output just fine.

The verification team's conda environment is avid_verify_py3, if you're able to access it.

Thanks in advance!
Molly

**Comment by jprestop** [comment]
Hi @mollybsmith-noaa.  We talked about this briefly at the METplus Engineering meeting today.  It sounds like @TatianaBurek is the best person to help with this task.  Unfortunately, @TatianaBurek is out until next week.  It was suggested that it would be good for you and @TatianaBurek to work on this together so that you both are familiar with this functionality.  

@TatianaBurek, upon your return, could you please coordinate a meeting with @mollybsmith-noaa?

**Comment by TatianaBurek** [comment]
@mollybsmith-noaa , I don't have an access to Jet and can't get the files and XML. I need them for the debugging. Could you please send them to me  or put them to some common location

**Comment by mollybsmith-noaa** [comment]
Jet is down for maintenance today but I’ll get them once it’s back up. Do you have access to any NOAA HPC machines or should I put the files in Google Drive or something?

**Comment by TatianaBurek** [comment]
I don't have an access to NOAA machines. Google Drive should work

**Comment by TatianaBurek** [comment, marked as answer]
The bug that prevents 'tcst' data loading was introduced last year. I created a bug report to address it:
https://github.com/dtcenter/METdataio/issues/256
